src/components/Etherscan/EtherscanLink.js

```diff
@@ -1,15 +1,14 @@
 import PropTypes from 'prop-types'
 import { network } from '../../environment'
+import { blockExplorerUrl } from '@aragon/ui'
 
-const { etherscanBaseUrl } = network
+const networkType = network.type
 
 // Render props component that injects an appropriate Etherscan url if possible
-const EtherscanLink = ({ address, children }) =>
-  children(
-    typeof etherscanBaseUrl === 'string'
-      ? `${etherscanBaseUrl}/address/${address}`
-      : null
-  )
+const EtherscanLink = ({ address, children }) => {
+  const etherscanUrl = blockExplorerUrl('address', address, { networkType })
+  return children(typeof etherscanUrl === 'string' ? etherscanUrl : null)
```

Comment:
We should be careful here with what we return, since `blockExplorerUrl()` will either return the url or an empty string (both fulfilling the `typeof etherscanUrl === 'string'` check).

Given `blockExplorerUrl()`'s behaviour, and how an empty string is more useful than `null` in `href` attributes, I think we can break this component's old API contract slightly and just call children with `etherscanUrl` immediately. The only component currently using `EtherscanLink` just uses a truthy check on this render prop, so this change won't affect it.